spark/src/main/kotlin/com/adevinta/spark/components/image/Image.kt

```diff
@@ -66,7 +66,7 @@ import com.adevinta.spark.tools.modifiers.sparkUsageOverlay
 
 @InternalSparkApi
 @Composable
-internal fun SparkImage(
+public fun SparkImage(
```

Comment:
Can we keep it internal and add a flag to use it in the configurator?

Reply:
Oh I was wandering how we can do it without triggering a compiler warning 🤔 
Do you have another solution than https://publicobject.com/2024/01/30/internal-visibility/ ?

I tried to use it but unfortunatly the compiler give me this warning

> This code uses error suppression for 'INVISIBLE_REFERENCE'. While it might compile and work, the compiler behavior is UNSPECIFIED and WON'T BE PRESERVED. Please report your use case to the Kotlin issue tracker instead: https://kotl.in/issue

Reply:
This was the solution I was going to give you. There might be additional compiler options to turn this warning off.
Otherwise, you can try the Annotation that restrict usage to a specific library (I think AndroidX uses this a lot, but I don't remember the name of this annotation).

Reply:
Should be resolved in e46e6116208ab87c0a0372dab5223b1689751000